readme.md

```diff
@@ -191,29 +191,55 @@ Search parameters to include in the request URL. Setting this will override all
 
 Accepts any value supported by [`URLSearchParams()`](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/URLSearchParams).
 
-##### prefixUrl
```

Comment:
We need even clearer docs about the difference between `baseUrl` and `prefix` and when to use each. Maybe with some real-world example use-cases. Otherwise, it's going to be confusing for users and that ends up being a support burden for us.